### ISSUE TITLE
Cybran, Seraphim and UEF Air Factories have a significantly slower roll off time than Land Factories, or the Aeon Air Factory

### ISSUE THREAD

**Issue by Basilisk3** [FAForever/fa#5627]
The Aeon Air Factory has exactly the same roll off time as the Land Factories, but the other Air Factories are significantly slower than either the Aeon Air Factory, or the Land Factories.

I noticed this when testing #5625.

**Comment by Garanas** [comment]
You're right, but to fix this means the animation speed needs to be extremely fast. And that looks off 😃

**Comment by Ejsstiil** [comment]
Isn't this kind-of intentional by design? I mean that every race indeed has a differences and it should - look for example the racial/national differences at the game Age of Empires for example. Or any other game, where one race has advantage in building, other has it cheaper, you got the idea.

If we slowly unify all races then all it remains is just a different graphics.
